package.json

```diff
@@ -32,61 +32,31 @@
     "stylelint": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
```

Comment:
BTW if you want to make this stricter let me know. We should also change the version we test on CI to match the minimum one.

Reply:
I think that this is good enough.